tests/controls/listctrltest.cpp

```diff
@@ -151,6 +154,37 @@ void ListCtrlTestCase::ColumnClick()
 
     m_list->ClearAll();
 }
+
+void ListCtrlTestCase::SubitemRect()
+{
+    wxImageList *m_imageListNormal = new wxImageList(32, 32, true);
+#ifdef wxHAS_IMAGES_IN_RESOURCES
+    m_imageListNormal->Add( wxIcon( "sample", wxBITMAP_TYPE_ICO_RESOURCE ) );
+#else
+    m_imageListNormal->Add( wxIcon( "sample.xpm" ) );
+#endif
+    m_list->InsertColumn(0, "Column 0", wxLIST_FORMAT_LEFT, 60);
+    m_list->InsertColumn(1, "Column 1", wxLIST_FORMAT_LEFT, 60);
+    m_list->InsertColumn(2, "Column 2", wxLIST_FORMAT_LEFT, 60);
+    for ( int i = 0; i < 3; i++ )
+    {
+        long tmp1 = m_list->InsertItem(i, wxString::Format("This is item %d", i), 0);
+        m_list->SetItemData(tmp1, i);
+        m_list->SetItem(tmp1, i, wxString::Format("Column 1 item %d", i));
```

Comment:
This is brain-twistingly misleading. You're setting the label of column `i` here, overwriting it in the next line... I have no idea what is this code supposed to do, but it definitely doesn't do anything useful.